src/web/routes/application/postcode/postcode.js

```diff
@@ -1,3 +1,29 @@
+const request = require('request-promise')
+const { wrapError } = require('../common/formatters')
+const { transformOsPlacesApiResponse } = require('./adapters')
+
+const OS_PLACES_API_URL = 'https://api.ordnancesurvey.co.uk/places/v1/addresses/postcode'
+
+const behaviourForPost = (config) => async (req, res, next) => {
+  const { OS_PLACES_API_KEY } = config.environment
+  const { postcode } = req.body
+
+  try {
+    const addressLookupResults = await request({
```

Comment:
Can we specify a timeout?

Reply:
Yes I think so. What timeout do you think is appropriate?

Reply:
good question! 30 seconds at the _very_ longest, but maybe something like 10 seconds?

Reply:
I think maybe 5s? If OS places takes 10s to respond then page requests will back up as the API request blocks the response from our server (to the user).

Reply:
Added a 5s timeout - can update to 10s if you think that is better.

Reply:
5 seconds is good.